openslides/motions/models.py

```diff
@@ -697,6 +698,57 @@ def get_root_rest_element(self):
         return self.motion
 
 
+class MotionChangeRecommendation(RESTModelMixin, models.Model):
+    """
+    A MotionChangeRecommendation object saves change recommendations for a specific MotionVersion
+    """
+
+    access_permissions = MotionChangeRecommendationAccessPermissions()
+
+    motion_version = models.ForeignKey(
+        MotionVersion,
+        on_delete=models.CASCADE,
+        related_name='change_recommendations')
+    """The motion version to which the change recommendation belongs."""
+
+    type = models.PositiveIntegerField()
+    """ Insertion (1), Deletion (2) or Replacement (0) """
```

Comment:
I wonder what "insertation" together with line_from==23 and line_to==42 should mean. Do we get an inconsistent database here?

(I always tried to argue that we have only one type at all: replacement. Every change recommendation is a replacement. Either I replace "foo bar" by "foo baz bar" with looks like an insertation or I replace "foo bar" by "foo" which looks like a deletion or I replace "foo" by "bar" which looks like a replacement. From the server's point of view we do not need to save the redundant type information.)